server/lib/sanitize/sanitize.coffee

```diff
@@ -36,6 +36,8 @@ sanitizeParameter = (input, name, config, place, res)->
     if config.optional then return
     else throw error_.newMissing place, name
 
+  if parameter.validatedByModel then return
```

Comment:
if i understand well all this, is it the responsability of sanitization lib to check models validation?

Reply:
the opposite: this is a flag to not try to validate in the sanitization lib. The name could be changed to something more explicit maybe, like `noValidation`